crates/solidity/inputs/schema/grammar/04-statements/01-blocks/productions.yml

```
@@ -23,22 +23,39 @@
 
 - name: "Statement"
   kind: "Parser"
-  unversioned:
-    choice:
-      - reference: "Block"
-      - reference: "SimpleStatement"
-      - reference: "IfStatement"
-      - reference: "ForStatement"
-      - reference: "WhileStatement"
-      - reference: "DoWhileStatement"
-      - reference: "ContinueStatement"
-      - reference: "BreakStatement"
-      - reference: "TryStatement"
-      - reference: "ReturnStatement"
-      - reference: "EmitStatement"
-      - reference: "RevertStatement"
-      - reference: "DeleteStatement"
-      - reference: "AssemblyStatement"
+  versioned:
+    0.4.11:
+      choice:
+        - reference: "Block"
+        - reference: "SimpleStatement"
+        - reference: "IfStatement"
+        - reference: "ForStatement"
+        - reference: "WhileStatement"
+        - reference: "DoWhileStatement"
+        - reference: "ContinueStatement"
+        - reference: "BreakStatement"
+        - reference: "ReturnStatement"
+        - reference: "EmitStatement"
+        - reference: "RevertStatement"
+        - reference: "DeleteStatement"
+        - reference: "AssemblyStatement"
+    0.6.0:
+      # added: "try" statement
```

nit suggestion to make it easier to copy/search:

```suggestion
      # added: "TryStatement"
```

---

Same for others.